ibc-testkit/src/testapp/ibc/core/types.rs

```diff
@@ -834,9 +558,8 @@ where
             .expect("Never fails")
     }
 
-    pub fn query_latest_header(&self) -> Option<HostBlock> {
-        let block_ref = self.host_block(&self.latest_height());
-        block_ref.cloned()
+    pub fn query_latest_header(&self) -> Option<&H::Block> {
+        self.host_block(&self.latest_height())
     }
```

Comment:
Shouldn't this return a header type?

Reply:
Good catch. I forgot to rename this method.

Reply:
531c3c92